esrally/driver/runner.py

```diff
@@ -2482,6 +2483,26 @@ def __repr__(self, *args, **kwargs):
         return "sql"
 
 
+class FieldCaps(Runner):
+    """
+    Retrieve `the capabilities of fields among indices.
+    <https://www.elastic.co/guide/en/elasticsearch/reference/current/search-field-caps.html>` _.
+    """
+
+    async def __call__(self, es, params):
+        index = params.get("index", "_all")
+        fields = params.get("fields", "*")
+        body = params.get("body", {})
+        index_filter = params.get("index_filter")
+        if index_filter:
+            body["index_filter"] = index_filter
+        request_params = params.get("request-params")
+        await es.field_caps(index=index, body=body, fields=fields, params=request_params)
+
```

Comment:
We're missing the return value here. If you look at the runner class, all runner need to return the number of operations that were done. This is important for the summary and reported metrics.

Have you tested this operation in an actual track?

```suggestion
        await es.field_caps(index=index, body=body, fields=fields, params=request_params)
        
        return {"weight": 1, "unit": "ops", "success": True}
```

Can you also please change the tests so that they check that this was returned?

Reply:
What should we return here? We only need to measure the latency of these requests.

Reply:
For the record, we need to return success and number of operations, per the suggestion you applied.